viabel/optimization.py

```diff
@@ -165,8 +185,20 @@ def descent_direction(self, grad):
         return descent_dir
 
 
-class WindowedAdaGrad(StochasticGradientOptimizer):
+class WindowedAdagrad(StochasticGradientOptimizer):
     """Adam optimization method
```

Comment:
Description needs to be updated. Definitely describe how the descent direction is calculated since it is non-standard